projects/hslayers/src/assets/locales/en.json

```diff
@@ -569,7 +573,10 @@
       "2": "Unable to open raster file",
       "18": "Missing one or more Shape files",
       "50": "Error when normalizing raster file"
-    }
+    },
+    "layerSuccessfullyRemoved": "Layer successfully removed",
+    "allLaymanlayersSuccessfullyRemoved": "All Layman layers successfully removed",
+    "laymanDeleteRequest": "Layman delete layers request"
```

Comment:
Id suggest reformulating these because user doesnt necessarily need to know whats Layman